icicle/include/msm/msm.cuh

```diff
@@ -103,6 +106,7 @@ namespace msm {
       false, // are_results_on_device
       false, // is_big_triangle
       false, // is_async
+      false, // is_signed
```

Comment:
If this allows for using less memory AND is faster, why don't we enable is as a default?

Reply:
he mentioned that it only works for with the big-triangle accumulation that is slower compare to the other accumulation algorithm.
@vladfdp can you say how does big-triangle + signed compared to the other accumulation without signed optimization?